base/deprecated.jl

```diff
@@ -1571,8 +1571,9 @@ end
     function prompt(msg::AbstractString; default::AbstractString="", password::Bool=false)
         Base.depwarn(string(
             "`LibGit2.prompt(msg::AbstractString; default::AbstractString=\"\", password::Bool=false)` is deprecated, use ",
-            "`get(Base.prompt(msg, default=default, password=password), \"\")` instead."), :prompt)
-        Base.get(Base.prompt(msg, default=default, password=password), "")
+            "`result = Base.prompt(msg, default=default, password=password); result === nothing ? \"\" : result` instead."), :prompt)
```

Comment:
Yes, as I noted above it would make sense to have a convenience function for that (not sure what it would be exactly). But it can always be added later (contrary to API breaks).

Reply:
Sorry, I forgot to reread those last comments in this thread! If no good name is found before this gets merged, I think I would be in favor of using a non-exported temporary name, like `_get` or whatever, so that at least the pattern is captured (and easily replaced when the new name comes), and at least something is available for the code which will be written in the meantime.